src/promptflow-devkit/promptflow/_cli/_utils.py

```diff
@@ -409,6 +409,51 @@ def _try_delete_existing_run_record(run_name: str):
         pass
 
 
+def get_instance_results(path: Union[str, Path]) -> List[Dict]:
```

Comment:
may add some test later, guard the result line content is expected

Reply:
Good one, will add the test in next pr.